lib/ui/window.dart

```diff
@@ -618,6 +618,19 @@ class Window {
   Size get physicalSize => _physicalSize;
   Size _physicalSize = Size.zero;
 
+  /// The physical depth is the maximum elevation that the Window allows.
+  ///
+  /// Physical layers drawn at or above this elevation will have their elevation
+  /// clamped to this value. This can happen if the physical layer itself has
+  /// an elevation larger than available depth, or if some ancestor of the layer
+  /// causes it to have a cumulative elevation that is larger than the available
+  /// depth.
+  ///
+  /// The default value is [double.maxFinite], which is used for platforms that
+  /// do not specify a maximum elevation.
+  double get physicalDepth => _physicalDepth;
+  double _physicalDepth = double.maxFinite;
+
```

Comment:
is it worth mentioning that this only does anything on fuchsia today? I can imagine an Android developer arriving here and being very confused.

Reply:
Sure